canonical-websites/community.ubuntu.com
# Fix small screen menu alignment

## Done
Hot fixed the menu alignment in small screens so that it always aligned with the centre of the navigation banner.

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8019/](http://0.0.0.0:8019/))
- Check that the large screen navigation has not changed
- Shrink screen to small and see that the `Menu` link is now vertically centred
- Open the menu and see the Close is also centred.

## Issue / Card
Fixes https://github.com/canonical-websites/community.ubuntu.com/issues/2

## Screenshots

![ubuntu community - ubuntu community 1](https://user-images.githubusercontent.com/1413534/26919694-6ef90ff2-4c2e-11e7-9ae1-94d451d3f568.png)
